modules/rosa-hcp-vpc-terraform.adoc

```diff
@@ -23,11 +27,20 @@ $ git clone https://github.com/openshift-cs/terraform-vpc-example
 ----
 
 . Navigate to the created directory by running the following command:
+ifndef::disconnected-rosa[]
 +
 [source,terminal]
 ----
 $ cd terraform-vpc-example
 ----
+endif::disconnected-rosa[]
+ifdef::disconnected-rosa[]
++
+[source,terminal]
+----
+$ cd terraform-vpc-example/zero-egress
```

Comment:
For some reason I'm not seeing this in the preview: https://81456--ocpdocs-pr.netlify.app/openshift-rosa-hcp/latest/rosa_hcp/rosa-hcp-egress-lockdown-install.html#rosa-hcp-creating-vpc-disconnected-install